solution/backend/regulations/rss_feeds.py

```diff
@@ -0,0 +1,49 @@
+from resources.models import AbstractResource
+from django.contrib.syndication.views import Feed
+from dateutil import parser
+
+
+class ResourceFeed(Feed):
+    title = 'Federal Register documents RSS Feed'
```

Comment:
Should be "Resources RSS Feed" as this encompasses both Federal Register docs and Supplemental Content items.